docs/ref/contrib/admin/index.txt

```diff
@@ -963,6 +963,31 @@ subclass::
     See the default template provided by django (``admin/filter.html``) for
     a concrete example.
 
+.. attribute:: ModelAdmin.list_filter_defaults
+
+    .. versionadded:: 1.10
+
+    Set ``list_filter_defaults`` to specify defaults for the filters, when the
+    user has not specified a filter the default will automatically be used.
+
+    The value should be a dictionary with as key the list filter field and as
+    value the default.
+
+    If you want to specify a more dynamic value that depends on the request
+    you can use the :meth:`~ModelAdmin.get_list_filter_defaults` method.
+
+.. attribute:: ModelAdmin.list_filter_sentinel
```

Comment:
Did you have a use case in mind? If so, it might be helpful to state it. I guess it would be if there's a value called "_off_" in your data? I guess it's okay, but I don't see a scenario where the method version would be needed though.